sdk/keyvault/keyvault-secrets/src/index.ts

```diff
@@ -687,4 +688,19 @@ export class SecretsClient {
 
     return resultObject;
   }
+
+  private getDeletedSecretFromDeletedSecretBundle(deletedSecretBundle: DeletedSecretBundle): DeletedSecret {
```

Comment:
Curious - in getSecretFromSecretBundle we splat the secretBundle.attributes, but here we're not. Does deleted secrets work differently than normal secrets?

Reply:
No it doesn't! I was just making the code a bit leaner. What did I miss?

Reply:
Sorry, wrong terminology, I meant 'spread' instead of splat

Reply:
From: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

Reply:
You can see the spread of the attributes in the original function:

https://github.com/Azure/azure-sdk-for-js/blob/e344b2d6ad200c93000a3b8d0a24bc02ba9bea94/sdk/keyvault/keyvault-secrets/src/index.ts#L674-L687

Reply:
Addressed! Thank you!